crates/common_utils/src/id_type.rs

```diff
@@ -168,11 +167,11 @@ where
     }
 }
 
-impl<const MAX_LENGTH: u8, const MIN_LENGTH: u8> Display for LengthId<MAX_LENGTH, MIN_LENGTH> {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", self.0 .0)
-    }
-}
+// impl<const MAX_LENGTH: u8, const MIN_LENGTH: u8> Display for LengthId<MAX_LENGTH, MIN_LENGTH> {
+//     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+//         write!(f, "{}", self.0. 0)
+//     }
+// }
```

Comment:
can u remove this comment

Reply:
Will be done in this PR. https://github.com/juspay/hyperswitch/pull/5380